index.js

```diff
@@ -1,33 +1,68 @@
 var visit = require('unist-util-visit');
 
 function sentenceNewline(ast, file, preferred, done) {
-  visit(ast, 'text', function (node) {
-    var lastDotFoundIdx = -1;
+  var blacklist = [];
+  if (typeof preferred === 'object' && !('length' in preferred)) {
```

Comment:
`preferred` is the value of the `sentence-newline` configuration in the `.remarkrc` file. By default it should be `undefined`. Since the package might get more options in the future I designed the blacklist option as an object. So `blacklist` is currently the only supported key in that. I think all your tests use the same configuration which has a blacklist.Therefore the condition is always true. Maybe you can add another test which doesn't have a blacklist. That should get the coverage higher.